cfme/tests/webui/test_general_ui_manual.py

```diff
@@ -56,3 +56,27 @@ def test_notification_window_can_be_closed_by_clicking_x():
         title: Notification window can be closed by clicking 'x'
     """
     pass
+
+
+@pytest.mark.manual('manualonly')
+@pytest.mark.tier(1)
+def test_infrastructure_provider_left_panel_titles():
```

Comment:
Please check out existing automation in this area, `cfme/tests/cloud_infra_common/test_relationships.py`.

Will likely allow you to remove this case, as its already covered by automation.